.github/workflows/main.yml

```diff
@@ -35,6 +35,10 @@ jobs:
 
       - name: Run rename test
         run: scripts/rename
```

Comment:
can you include this new check in here?

Reply:
Sure. Though it's not really part of "rename", it's a general check to make sure the SDist isn't missing files. When the renaming goes away, this will would be useful. Would it be better in `scripts/check`?

Reply:
Unrelated question: would `scripts/lint` be better named `scripts/format`? `scripts/check` runs all the liners and `scripts/lint` runs the formatter.

Reply:
yeah, on check is good.

Reply:
`lint` follows the `encode` scripts I'm used.